config-template.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Honestly add this to your .gitignore file so that you don't accidentally leak your tokens. Better yet create a whole new `config-token.json` that you gitignore and use for any tokens that should be published

Reply:
Yep that's why this is a template, in the .gitignore is the config.json which contains the tokens